test/modules/image-sequencer.js

```diff
@@ -164,6 +164,12 @@ test('insertSteps({image: {index: index, name: "module", o: options} }) inserts
   t.end();
 });
 
+test('toString() and stepToString() return the step/steps in string format',function(t){
+  t.equal(sequencer.toString(),"channel(channel:green),channel(channel:green),channel(channel:green),channel(channel:green),channel(channel:green)","toString works");
+  t.equal(sequencer.stepToString(sequencer.steps[1]),"channel(channel:green)","stepToString works");
```

Comment:
Here, let's just use the previous version of `sequencer.images.image1.steps` until we more fully rewire how `sequencer.steps` works -- we want the tests to demonstrate the "proper" way to use things, and before we've fully implemented a `sequencer.steps` approach, let's let the tests show the old method. Make sense? Then we can switch this over.

Reply:
@jywarren Actually then we wont get the steps access in case of test since test uses an image by the name of test and otherwise image is image1 so either we would have to pass an extra parameter in case of test, what say??